src/Samples/SarifDeferredSample/SarifDeferredSample.csproj

```diff
@@ -34,8 +34,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Newtonsoft.Json.13.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\packages\Newtonsoft.Json.11.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
```

Comment:
>Newtonsoft.Json.11.0.2 [](http://example.com/codeflow?start=28&length=22)

to <Reference> version 11
need to have <PackageReference> for that version

<PackageReference Include="Newtonsoft.Json" Version="" />

Reply:
indeed, thanks, fixed up!